README.md

```diff
@@ -106,37 +106,35 @@ fail-on-scopes:
 
 ### allow-licenses
 
-Only allow the licenses in this list. See "[Licenses](https://github.com/actions/dependency-review-action#licenses)".
+Only allow the licenses that comply with the expressions in this list. See "[Licenses](https://github.com/actions/dependency-review-action#licenses)".
```

Comment:
```suggestion
Only allow the licenses in this list. The license name(s) should be SDPX-compliant. See "[Licenses](https://github.com/actions/dependency-review-action#licenses)".
```

Small reword, justification: if my repo contains code/assets under two licenses then my repo's/project's license is `MIT AND GPL-2`. Although we refer to this as an "expression" when parsing, for the end user every one of these expressions is a license by itself. A user should not be writing SPDX expressions by hand since likely no licenses in the SPDX database will match custom made patterns.